src/Symfony/Component/Cache/CacheItem.php

```diff
@@ -22,6 +22,7 @@
 final class CacheItem implements ItemInterface
 {
     private const METADATA_EXPIRY_OFFSET = 1527506807;
+    public const RESERVED_CHARACTERS = '{}()/\@:';
```

Comment:
public before private?

Reply:
I wasn't sure about this, because the private const was before protected properties. Unless all consts are supposed to be before properties regardless of visibility.

Reply:
Yes, constants before properties before methods.